sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -201,33 +201,32 @@ public DirectConnectionConfig setMaxRequestsPerConnection(int maxRequestsPerConn
      *
      * @return the request timeout interval
      */
-    public Duration getRequestTimeout() {
+    Duration getRequestTimeout() {
         return requestTimeout;
     }
 
     /**
      * Sets the request timeout interval
      * This represents the timeout interval for requests
      *
-     * Default value is 60 seconds
+     * Default value is 5 seconds
      *
      * @param requestTimeout the request timeout interval
      * @return the {@link DirectConnectionConfig}
      */
-    public DirectConnectionConfig setRequestTimeout(Duration requestTimeout) {
+    DirectConnectionConfig setRequestTimeout(Duration requestTimeout) {
```

Comment:
does consistency stack use this timeout?

Reply:
From my current understanding. the consistency stack is not using it. 

For the request happens the first time, it is using 60s which comes from InitialArgumentValuePolicyArg = Quadruple.with(false, false,Duration.ofSeconds(60), 0);